### PR TITLE
Add GDCR18 in Vechta

### PR DESCRIPTION
**Pull request by StefanMacke**
Add GDCR18 in Vechta/Germany to the list of events.

### CODE REVIEW

File: _data/events/Vechta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GDCR18 in Vechta",
+  "url": "https://doodle.com/poll/v3795putggfv68ab",
```
Comment (thodorisbais) [inline _data/events/Vechta.json]: I am not so fond of using doodle for this kind of purpose. Is there any chance we could use some other mean (i.e. meetup.com, eventbrite or something similar)?

**Comment by StefanMacke** [comment]
I've moved the JSON file to the right directory and it shows up on the preview page.

File: _data/events/Vechta.json (hunk 1 above)
Comment (thodorisbais) [reply _data/events/Vechta.json]: We have a deal! Thanks!

**Comment by thodorisbais** [comment]
Welcome @StefanMacke , have a good one!

**Pull request merged**